docs/quickstart.md

```
@@ -0,0 +1,248 @@
+```bash
+# Clone the repository
+git clone https://github.com/BumpyClock/digests-api.git
+cd digests-api
+
+# Build and run with Docker
+docker build -t digests-api .
+docker run -p 8000:8000 -e PORT=8000 digests-api
+```
+
+**Test it works:**
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)